zds/utils/templatetags/tests/tests_date.py

```diff
@@ -109,9 +109,10 @@ def test_humane_time(self):
                       "{{ date_epoch | humane_time }}"
                       ).render(self.context)
 
-        # Since ZdS is in Europe/Paris, hours can be 1 or 2
-        is_dst = time.daylight and time.localtime().tm_isdst > 0
-        utc_offset_sec = time.altzone if is_dst else time.timezone
-        hours = -1 * utc_offset_sec / 3600
+        # Since ZdS is in Europe/Paris, hours can be 0 or 1
+        paris = timezone.pytz.timezone('Europe/Paris')
```

Comment:
Comment c'est possible qu'on ait des problèmes avec le changement d'heure
que maintenant ?
Le 28 mars 2016 09:50, "Quentin Pradet" <notifications@github.com> a écrit :

> In zds/utils/templatetags/tests/tests_date.py
> <https://github.com/zestedesavoir/zds-site/pull/3468#discussion_r57552365>
> :
>
> > @@ -109,9 +109,10 @@ def test_humane_time(self):
> >                        "{{ date_epoch | humane_time }}"
> >                        ).render(self.context)
> >
> > -        # Since ZdS is in Europe/Paris, hours can be 1 or 2
> > -        is_dst = time.daylight and time.localtime().tm_isdst > 0
> > -        utc_offset_sec = time.altzone if is_dst else time.timezone
> > -        hours = -1 * utc_offset_sec / 3600
> > +        # Since ZdS is in Europe/Paris, hours can be 0 or 1
> > +        paris = timezone.pytz.timezone('Europe/Paris')
>
> victor: ok, merci (c'est peut-être quelque chose à activer mais passons)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3468/files/d4531a269fba87216a531c432198acabb5751d09#r57552365>
>